ccore/include/pyclustering/interface/elbow_interface.h

```diff
@@ -33,43 +33,45 @@
 #include <pyclustering/definitions.hpp>
 
 
-/**
- *
- * @brief   Elbow result is returned by pyclustering_package that consist sub-packages and this enumerator provides
- *           named indexes for sub-packages.
- *
- */
+/*!
+
+@brief   Elbow result is returned by pyclustering_package that consist sub-packages and this enumerator provides
+          named indexes for sub-packages.
+
+*/
 enum elbow_package_indexer {
     ELBOW_PACKAGE_AMOUNT = 0,
     ELBOW_PACKAGE_WCE,
     ELBOW_PACKAGE_SIZE
 };
 
 
-/**
- *
- * @brief   Performs data analysis using Elbow method using center initializer that is specified by template.
- * @details Caller should destroy returned result by 'free_pyclustering_package'.
- *
- * @param[in] p_sample: input data for clustering.
- * @param[in] p_kmin: minimum amount of clusters that should be considered.
- * @param[in] p_kmax: maximum amount of clusters that should be considered.
- * @param[in] p_random_state: seed for random state (by default is `RANDOM_STATE_CURRENT_TIME`, current system time is used).
- *
- * @return  Returns Elbow's analysis results as a pyclustering package [ [ amount of clusters ], [ within cluster errors (wce) ] ].
- *
- */
+/*!
+
+@brief   Performs data analysis using Elbow method using center initializer that is specified by template.
```

Comment:
- [ ] Update to: `Performs data analysis using Elbow method and center initializer...`